content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -61,74 +75,78 @@ To combine an event, click the three dots next to an event and select **+ Combin
 
 Use the Group by dropdown to group the data by a specific attribute.
 
-**Note**: `Group by` does not work with the funnel steps visualization, your visualization automatically changes to a top list.
+**Note**: The `Group by` option is not supported for the funnel steps visualization; when applied, the visualization automatically changes to a top list.
 
 ## Refine conversion
 
 You can further analyze the information on the funnel page to understand the conversion rate. Conversion rate is a crucial metric that measures the effectiveness of your site or application.
 
-You can analyze conversion by **session**, **users** or **accounts**, which means you can understand conversion across all sessions, or by distinct users or accounts. This can be useful if you suspect, for instance, that a minority of your user base converts at a high rate.
+You can analyze conversion by **session**, **users** or **accounts**. This can be useful if you suspect, for instance, that a minority of your user base converts at a high rate.
 
-{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="Specify which facet you'd like to use to refine your conversion. The available options are Session, User, and Account." style="width:50%;" >}}
+- If you select <strong>Session</strong>, all steps must be completed within the same <code>@session.id</code> to count as a conversion.
 
+- If you select <strong>User</strong>, the funnel requires the same individual user (<code>@user.id</code>) to complete every step for the conversion to count.
 
-<div class="alert alert-info"> 
+- If you select <strong>Account</strong>, different users within the same account can complete different steps and the conversion still counts. In this case, the funnel is tied to the <code>@account.id</code> facet.
 
-If you select <code>Session</code>, all steps must be completed within the same <code>@session.id</code> to count as a conversion.
-
-If you select <code>User</code>, the funnel requires the same individual user (<code>@user.id</code>) to complete every step for the conversion to count.
+{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="Specify which facet you'd like to use to refine your conversion. The available options are Session, User, and Account." style="width:50%;" >}}
```

Comment:
```suggestion
{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="The section of the UI where you can select Session, User, or Account to analyze your conversion." style="width:50%;" >}}
```
I would make alt text a description of what the screenshot is, rather than providing an instruction. Alt text exists to provide an equivalent experience for users who can't see the image (due to things like visual impairments or internet connectivity issues), so all you have to do is close that gap and you're set! A good way of thinking about this is imagining you're telling a friend on the phone about it 🙂